RELEASE.rst

```diff
@@ -0,0 +1,10 @@
+Release Instructions
+====================
+
+1. Prepare master branch for release (make sure all PRs are merged and tests pass).
```

Comment:
Does every merge to master mean an increase in minor version number? How do we decide when to increase version number? (This is probably a better discussion for offline, but maybe we could add some guidelines to our code review document?)

Reply:
No, you have to manually check out the master branch and perform this release process to increase the version number. I think it's a separate thing from code reviews, but we could have more discussion about our overall code design process. If it's plugged into our current process, we'd discuss this in the morning or during our weekly meetings.